airflow/lineage/backend/__init__.py

```diff
@@ -16,13 +16,12 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-"""Sends lineage metadata to a backend"""
+"""Send lineage metadata to a backend"""
 
 
 class LineageBackend:
-    """Sends lineage metadata to a backend"""
-    def send_lineage(self,
-                     operator=None, inlets=None, outlets=None, context=None):
+    """Sends lineage metadata to a backend."""
+    def send_lineage(self, operator, inlets, outlets, context):
```

Comment:
Why did we change it from Optional to Required?

Reply:
I imagine Pylint was complaining because the subclass had a different signature? If so the change should be to the subclass to make them optional, not removing the optional here.

Reply:
> If so the change should be to the subclass to make them optional, not removing the optional here.

True - that would also not break backwards compatibility.